modules/lofreq/1.0/lofreq.smk

```diff
@@ -73,6 +75,9 @@ rule _lofreq_run:
         **CFG["resources"]["lofreq"]
     shell:
         op.as_one_line("""
+        PATH={SCRIPT_PATH}:$PATH;
+        SCRIPT=$(which lofreq2_call_pparallel.py);
```

Comment:
I wonder if using the path to the script instead of `which` here would help ensure it's not inadvertently using the Conda-installed version. Then you could include something like `if [[ $SCRIPT == "" ]]; then echo "Lofreq script not found in module directory. Please ensure you are using the latest version from Master.; fi`

Reply:
I'm not sure I understand. How would I use the path to the script? Wouldn't this check only work if the user is already using the latest version?

Reply:
I understand now. That would indeed protect against any unexpected environment issues. In theory prepending to PATH should make this check unnecessary but from recent experience, it's better not to assume this will always work.